rules/windows/defense_evasion_network_connection_from_windows_binary.toml

```diff
@@ -111,11 +113,14 @@ tags = [
     "Resources: Investigation Guide",
     "Data Source: Elastic Defend",
     "Data Source: Sysmon",
+    "Data Source: SentinelOne",
+    "Data Source: Crowdstrike",
```

Comment:
```suggestion
```
CS is inconsistent with process.entity_id for now, network events don't have it yet (this is documented and pending fix), and it don't populate process.parent.executable